week-9/Homework/mandatory/2-exercises/4-music-player.js

```diff
@@ -21,8 +21,38 @@ This means the order the songs are played in will be random, but each song will
 
 class MusicPlayer {
     // Add your code here
+    constructor(){
+       this.index
+       this.music = []
+    }
+add(song, singer){
+let addSong = {
+    song: song,
+   singer: singer
+}
+this.music.push(addSong)
+}
+play(){
+    this.index = 0
+    if(this.music.length > 0 ){
```

Comment:
Will this ever be true if you always set index to be 0 on line 36? :)

Reply:
I think it might be true even without giving a value :|

Reply:
Sorry my comment is incorrect again!